packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -40,6 +40,12 @@ export class VaBreadcrumbs {
    *  Represents a list of breadcrumbs. Use an array of objects with label and href properties, and then use JSON.stringify() to convert to a string. This prop is available when `uswds` is set to `true`.
    */
   @Prop() breadcrumbList?: any;
+
+  /**
+   *  When true, the first breadcrumb label will be "VA.gov home". 
+   */
+  @Prop() homeVeteransAffairs?: boolean = true;
```

Comment:
Naming is hard. I originally had this as `homeVA` but with the capitalization, the prop would need to be set like `home-v-a` on the vanilla web component which is ick.

Then I thought about just doing `homeVa` with the lowercase "a" but I didn't want it to look like we were not using the correct VA branding.

I landed on this `homeVeteransAffairs` because it never hurts to be more explicit about what this prop does and we can avoid the capitalization confusion.